root/opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/systemd.rb

```diff
@@ -22,9 +22,15 @@
     confine osfamily: 'RedHat'
     setcode do
         systemd = {
-            "restart" => {}
+            "restart" => {},
+            "error" => ""
```

Comment:
Why catch the error? Is it really usefull?

Reply:
Since every error is caught and silenced I need a place where to store that information.

You make me think to catch only the timeout error and leave any other possible error to STDERR... :thinking: